app/views/layouts/application.html.erb

```diff
@@ -117,6 +117,7 @@
                                     <ul id="nav">
                                         <li><%= link_to "Home", "/" %></li>
                                         <% if user_signed_in? %>
+                                            <li><%= current_user.role %></li>
```

Comment:
Is there a reason we need the role to show up here?

Reply:
Sorry! That was for myself - forgot to remove it.